lib/World.lua

```diff
@@ -625,6 +625,124 @@ function QueryResult:without(...)
 	return self
 end
 
+--[=[
+	@class View
+
+	Provides random access to the results of a query.
+
+	Calling the View is equivalent iterating a query. 
```

Comment:
There's a typo here, but I'm also not sure if it's a good idea to include this. I think we're going to have issues motivating views vs. queries to people and so maybe highlighting differences is better